runtime/src/swam/runtime/config/EngineConfiguration.scala

```diff
@@ -59,14 +59,14 @@ case class LowLevelStackConfiguration(size: Information)
 
 case class HighLevelStackConfiguration(size: Information, callDepth: Int)
 
-case class TraceConfiguration(handler: HandlerType,
+case class TraceConfiguration(var handler: HandlerType,
```

Comment:
Why is a `var` needed and not a `val`? Prefer immutable values everywhere. #FunctionalProgrammingRocks

Reply:
Well...there is no need. I was using it to change it in the Test suite for the tracer. Thanks again !! :)